packages/shared/src/styles/base.css

```diff
@@ -32,6 +32,8 @@ body {
       padding-right: 0.5rem;
     }
   }
+  
+  overflow-x: hidden;
```

Comment:
We never had a horizontal display that requires the user to scroll the page. Applying this property because without this the horizontal scroll is resulting in unnecessary jumpyness.

Preview without this:

https://user-images.githubusercontent.com/13744167/219273409-92f03934-2f8a-4706-8060-cd02233621fd.mov